src/main.cpp

```diff
@@ -530,7 +552,26 @@ LRESULT WINAPI WndProc(HWND hWnd, UINT msg, WPARAM wParam, LPARAM lParam)
 			SetFocus(hWnd);                 // Set the focus on the window.
 			SetForegroundWindow(hWnd);      // And bring it to the foreground.
 		}
+		
+		if (lParam ==  WM_RBUTTONDOWN &&     // Right click on notification icon.
+			gApp.mNotifMenuHmenu == nullptr) // Create the menu only if it's not already open.
+		{
+			// Create the menu where the mouse is placed.
+			POINT cursorPosition = {};
+			BOOL ret = GetCursorPos(&cursorPosition);
+			gAssert(ret);
+
+			HMENU hMenu = CreatePopupMenu();
+			gApp.mNotifMenuHmenu = hMenu;
+			ret = InsertMenu(hMenu, 0, MF_BYPOSITION | MF_STRING, cNotifMenuCookErrored, L"Cook errored");
```

Comment:
In the rest of the code I've tried to use the A version of the Windows functions instead of the WCHAR ones. They should support receiving utf-8 strings thanks to the `enable_utf8.manifest` (but does need checking, because some of then don't work!)

Just try `InsertMenuA(..., u8"Cook errored 😭")` and see what happens :D

Reply:
My bad, I noticed usage of W function in the same file so I went with the flow. I'm going to change them then, thanks for pointing at it.